docs/source/blog.rst

```diff
@@ -0,0 +1,69 @@
+Blog
+====
+
+We publish on `blog.jupyter.org` using the Ghost blog platform. Here are some
+guideline and warnings for authors using our blog platform either regularly, or
+as guests. 
+
+If you have any doubt, ask someone that have already published, some things
+cannot be easily undone. 
+
+You want to publish something on our blog ? Feel free to ask us, tell us what
+you want to write about. Hope fully it's Jupyter related ! We are always happy
+to get new content. If you don't feel comfortable writing on your own, we can
+alway help you to write you post. 
+
```

Comment:
I think this little paragraph should actually be at the top of the document so that people know what the rest of the information on the page is relevant to those who want to publish something on the Jupyter blog.